### PR TITLE
Create dx.yaml

### PR DESCRIPTION
**Pull request by tylerwray**
## Hey, I just made a Pull Request!

Adding DX Plugin to the plugins directory

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by tylerwray** [comment]
Keep alive please?

File: microsite/data/plugins/dx.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: DX
+author: DX
+authorUrl: https::/getdx.com
```
Comment (benjdlambert) [inline microsite/data/plugins/dx.yaml:4]: Think this might need an update :)

**Review by benjdlambert — COMMENTED** [review]
Just a small nit.

Also wondering if we should be listing plugins in the plugins directory that are in closed beta, and aren't available to everyone to use yet?

File: microsite/data/plugins/dx.yaml (hunk 1 above)
Comment (tylerwray) [reply microsite/data/plugins/dx.yaml:4]: Thank you! I'll fix haha 

And I'll go update our README's, we are no longer in a closed beta 👍🏻

Comment (tylerwray) [reply microsite/data/plugins/dx.yaml]: @benjdlambert I've removed those beta warnings from our package README 👍🏻

**Review by benjdlambert — APPROVED** [review]

**Comment by benjdlambert** [comment]
Nice thanks! :tada:

**Pull request merged**